src/contributors/judithparedes.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Judith Mishel Paredes Artica", 
+    "desc": "Front End Developer",
+    "github" : "https://github.com/mishelpa",
+    "twitter" : "https://twitter.com/MishelJudith", 
```

```suggestion
    "twitter": "https://twitter.com/MishelJudith", 
```
Remove unwanted `white-space`.